codeeditor demo running panic

error message
```
> go run .\main.go
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x28 pc=0x7ff790be28ff]

goroutine 1 [running, locked to thread]:
github.com/AllenDang/giu.(*context).GetWidgetIndex(...)
        D:/Go/pkg/mod/github.com/!allen!dang/giu@v0.6.3-0.20230807193504-b8dd4052ada8/Context.go:167
github.com/AllenDang/giu.GenAutoID(...)
        D:/Go/pkg/mod/github.com/!allen!dang/giu@v0.6.3-0.20230807193504-b8dd4052ada8/Widgets.go:12 
github.com/AllenDang/giu.CodeEditor()
        D:/Go/pkg/mod/github.com/!allen!dang/giu@v0.6.3-0.20230807193504-b8dd4052ada8/CodeEditor.go:45 +0x1f
main.main()
        D:/Go/src/giu-demo/main.go:52 +0x3f
exit status 2
````

--- a/examples/codeeditor/codeeditor.go
+++ b/examples/codeeditor/codeeditor.go
@@ -47,6 +47,8 @@ func loop() {
 }
 
 func main() {
+	wnd := g.NewMasterWindow("Code Editor", 800, 600, 0)
+
 	errMarkers = imgui.NewErrorMarkers()
 
 	editor = g.CodeEditor().
@@ -56,6 +58,5 @@ func main() {
 		LanguageDefinition(giu.LanguageDefinitionSQL).
 		Border(true)
 
-	wnd := g.NewMasterWindow("Code Editor", 800, 600, 0)
 	wnd.Run(loop)
 }

